extensions/resource-deployment/src/services/tools/kubectlTool.ts

```diff
@@ -13,7 +13,7 @@ export class KubeCtlTool implements ITool {
 	}
 
 	get description(): string {
-		return localize('resourceDeployment.KubeCtlDescription', 'Tool used for managing the Kubernetes cluster');
+		return localize('resourceDeployment.KubeCtlDescription', 'The command-line tool allows you to run commands against Kubernetes clusters');
```

Comment:
I'd sync with PMs on phrasing but this reads a little oddly. Would suggest something like

"A command-line tool that enables running commands against Kubernetes clusters."